danger.gemspec

```diff
@@ -25,6 +25,7 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency 'colored'
   spec.add_runtime_dependency 'nap'
   spec.add_runtime_dependency 'octokit'
+  spec.add_runtime_dependency 'rest-client'
```

Comment:
sure, I'll take a look at that later. I hadn't used rest-client before also, just seen some gem using it and thought it was good enough.